shishir/Day1/activity_main.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
ID is used to reference that view to perform actions just like you do with any other element. Although id is not used anywhere in this code, it is always a good practice to assign ids to all parent and child elements. Make changes to other properties that I pointed out to align the views properly.